src/server/service/kv.rs

```diff
@@ -677,6 +685,41 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
         ctx.spawn(future);
     }
 
+    fn raw_batch_get(
+        &self,
+        ctx: RpcContext,
+        mut req: RawBatchGetRequest,
+        sink: UnarySink<RawBatchGetResponse>,
+    ) {
+        const LABEL: &str = "raw_batch_get";
+        let timer = self.metrics.raw_batch_get.start_coarse_timer();
+
+        let keys = req.get_keys()
+            .into_iter()
+            .map(|x| Key::from_raw(x))
```

Comment:
For raw key, it doesn't need encoding.

Reply:
Sure, I must be copying skeleton code from kv interface. Let me fix it, thanks.

Reply:
Oh, I remember why I did it like this now. Inside storage later, the snapshot interface expects Key, so it's just a matter of doing the conversion at where. So I'm moving the conversion to storage layer, will update this pr soon.